launchers/junit/build.gradle.kts

```diff
@@ -34,8 +34,8 @@ dependencies {
 
 publishing {
```

Comment:
Seems like this is the only launcher with needs to publishing. I'm not sure that we should consider this as a launcher because I think it fit better as a test utils package. The name should be `junit` or `junit-extension`.
I know that we already had this discussion.

Reply:
@ndr-brt This should be separate from test utils which are internal classes the EDC uses for testing, whereas this is a JUnit extension accessed by users. I'm happy to call it junit-extension, but it should still be a launcher since it bootstraps a runtime as part of a JUnit test fixture.

Reply:
Ok, but what about `BaseRuntime`? It's a runtime but it's contained in the `core:bootstrap` module.
For the release process we should find a clear separation between what should be published and what should be not.
We could separate the code (`EdcExtension` and eventually other test utils classes) from the launcher itself (the gradle file) as we did for the `BaseRuntime` that is separated from `basic` launcher. 
That will permit us to state that "no launcher has to be published"

Reply:
will change it to `junit-extension`. we can figure out whether we want to move this to a different subfolder later.